htdocs/install/mysql/migration/19.0.0-20.0.0.sql

```diff
@@ -0,0 +1 @@
+UPDATE llx_c_units SET scale = 1 WHERE code = "S";
```

Comment:
SQL stringmust use simple quotes so   = 'S';
This will solve the CTI error about SQL syntax...

Reply:
Done